OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -632,7 +665,7 @@ function Get-PublicHostName
     return $publicHostName
 }
 
-function Get-TentacleDownloadUrl
+function Get-TentacleDownloadUrl # can we validate the specified installer too? # should we?
```

Comment:
I'm wondering if we can validate the actual URL provided for TentacleDownloadUrl and TentacleDownloadurl64, because I've specified the wrong URL for 64-bit before now.

Reply:
Not sure we can easily.

Reply:
I'll pull the comment for now